guides/source/active_support_instrumentation.md

```diff
@@ -686,6 +686,12 @@ INFO. The only ActiveStorage service that provides this hook so far is GCS.
 | ------------ | ------------------- |
 | `:key`       | Secure token        |
 
+### transform.active_storage
+
+| Key          | Value               |
+| ------------ | ------------------- |
+| `:service`   | Name of the service |
```

Comment:
This key isn't included in the payload, as far as I can tell:

https://github.com/rails/rails/blob/162f96917cdb9755b06bb6c79a949bd00da75907/activestorage/app/models/active_storage/variation.rb#L55

Reply:
True. Thanks for pointing out.